extensions/Worker.Extensions.CosmosDB/release_notes.md

```diff
@@ -4,6 +4,7 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.CosmosDB <version>
+### Microsoft.Azure.Functions.Worker.Extensions.CosmosDB 4.10.0
+
+- Updating dependencies and webJobs extensions reference to 4.7.0
```

Comment:
Can you please add the dependencies to the list as well?

Reply:
Updated at all the places.